deployment/deploy/DeployTask.cpp

```diff
@@ -1635,7 +1635,7 @@ class CDeployTask : public CInterface, implements IDeployTask
        {
          retcode = 5;
          errbuf.clear().append("Invalid or missing key passphrase. ");
-         pipe->abort();
+         pipe->abort(false);
```

Comment:
I suspect this/almost all should be graceful too really, but to avoid potential deadlock from misbehaving child, it would be good if abort fired a SIGKILL after a given period of grace time..